src/ripple/app/tx/impl/CreateCheck.cpp

```diff
@@ -186,7 +186,8 @@ CreateCheck::doApply()
     // Note that we we use the value from the sequence or ticket as the
     // Check sequence.  For more explanation see comments in SeqProxy.h.
     std::uint32_t const seq = ctx_.tx.getSeqProxy().value();
-    auto sleCheck = std::make_shared<SLE>(keylet::check(account_, seq));
+    Keylet const checkKeylet = keylet::check(account_, seq);
```

Comment:
Since you are in the neighborhood, please correct the comment by removing the extra 'we'.